packages/ui/src/machines/authenticator/defaultServices.ts

```diff
@@ -169,7 +169,7 @@ export const defaultServices = {
     }
   },
   async validatePreferredUsername(
-    formData: AuthFormData,
-    touchData: AuthTouchData
+    _: AuthFormData,
+    __: AuthTouchData
```

Comment:
Linting fix due to new rules ([ref](https://github.com/aws-amplify/amplify-ui/pull/5830/files#diff-6e97cd3294540e00be8979d5d20d4f092e2d49d3d7e796d472f3e03961d69ab5R31))